Explore: Add what's new about panel plugins (#72338)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v10-1.md

For Elasticsearch metric queries in Explore, you can now see the sample of log lines that contributed to the displayed results. To see these logs, click on the collapsed Logs sample panel under your graph or table panel. If you want to interact with your log lines or modify the log query, click on the “Open logs in split view” button and the log query will be executed in the split view.

### Panel plugins in Explore

<!-- Ben Donnelly -->

_Experimental in all editions of Grafana_

> Data source plugin developers can now use any plugin to visualize the data in Explore. Similarly to `preferredVisualizationType` we have introduced an experimental API to render visualization by plugin ID. In the returned data frame, set meta option `preferredVisualisationPluginId` to a plugin ID you want to be used when showing the data for given data frame.

Reviewer: ```suggestion
Data source plugin developers can now use any plugin to visualize data in Explore. Similar to `preferredVisualizationType`, we've introduced an experimental API to render visualizations by plugin ID. In the returned data frame, set the meta option `preferredVisualisationPluginId` to the plugin ID you want to be used when showing the data for given data frame.
```
